Circle CI is failing for master

https://circleci.com/gh/minishift/minishift/5838
```
#!/bin/bash -eo pipefail
cd $GOPATH/src/github.com/minishift/minishift && make prerelease

^D^Drm -rf /Users/distiller/go/pkg/darwin_amd64/github.com/minishift
rm -rf out
rm -rf release
rm -f  docs/source/_tmp/*.md
cmd/minishift/cmd/config/config_test.go
cmd/minishift/cmd/config/view_test.go
pkg/minishift/docker/image/oci_image_handler.go
pkg/util/crypto.go
make: *** [fmtcheck] Error 1
Exited with code 2
```

However I can not reproduce this locally with same go version. Also it had passed for the PR.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -64,7 +64,7 @@ jobs:
     macos:
       xcode: "9.0"
     environment:
-        GOVERSION: 1.10.2
+        GOVERSION: 1.11.2
         XCODE_SCHEME: symbolicator
         XCODE_WORKSPACE: symbolicator
 

